AppCenter/AppCenter/MSConstants.h

```diff
@@ -70,12 +70,15 @@ typedef NS_ENUM(NSInteger, MSInitializationPriority) {
 /**
  * Enum with the different HTTP status codes.
  */
-typedef NS_ENUM(NSInteger, MSHTTPCodesNo) { // Informational
+typedef NS_ENUM(NSInteger, MSHTTPCodesNo) { // Invalid
```

Comment:
The comment should be in the next line.

Reply:
Not true according to your dear formatter ;)

Reply:
Found a work around!